library/std/src/sys/pal/unix/fd.rs

```diff
@@ -313,6 +314,7 @@ impl FileDesc {
         cfg!(not(any(target_os = "espidf", target_os = "horizon", target_os = "vita")))
     }
 
+    #[allow(unused_unsafe)]
```

Comment:
I don't want to make this allowance for OS that don't need it.
```suggestion
    #[cfg_attr(target_os = "vxworks", allow(unused_unsafe)]
```